### ISSUE TITLE
NLLB-200 support

### ISSUE THREAD

**Issue by kaspar030** [ggerganov/ggml#137]
It would be great to have ggml support for Facebook's No Language Left Behind 200x200 translation model:

https://ai.facebook.com/research/no-language-left-behind/

https://huggingface.co/facebook/nllb-200-3.3B

**Comment by guranu** [comment]
> It would be great to have ggml support for Facebook's No Language Left Behind 200x200 translation model:
> 
> https://ai.facebook.com/research/no-language-left-behind/
> 
> https://huggingface.co/facebook/nllb-200-3.3B

Is it possible to convert it to ggml now?

**Comment by zhangp365** [comment]
NLLB-200 is a great model. It would be wonderful if GGML could support this model.

**Comment by bil-ash** [comment]
Yes, I would also like to use ggml for inferencing nllb models. @ggerganov Is conversion/inference possible as of today? If not any ETA?

**Comment by ggerganov** [comment]
Try looking for community help in `llama.cpp` - we've already implemented several architectures there
